src/status_im/ui/screens/wallet/send/events.cljs

```diff
@@ -202,8 +208,8 @@
    ;; for the recipient, we always redirect to `:wallet-transaction-sent` even when we don't
    (let [send-command? (and chat-id (get-in db [:id->command ["send" #{:personal-chats}]]))]
      (fx/merge cofx
-               #(when send-command?
-                  (commands-sending/send % chat-id send-command? params))
+               #_#(when send-command?
```

Comment:
Should this code be removed?

Reply:
No, it should just be re-enabled to send the chat message (Anna's point number 2)

Reply:
And I re-enabled it. Could you re-review?

Reply:
on it